.circleci/config.yml

```diff
@@ -535,6 +538,13 @@ step-fix-known-hosts-linux: &step-fix-known-hosts-linux
         ./src/electron/.circleci/fix-known-hosts.sh
       fi
 
+step-ninja-summary: &step-ninja-summary
+  run:
+    name: Print ninja summary
+    command: |
+      cd src
+      python third_party/depot_tools/post_build_ninja_summary.py -C out/Default
```

Comment:
any particular reason to use the vendored depot_tools instead of the one that's installed on CI?

Reply:
I saw this comment and thought it was always being checked out to third_party: https://github.com/electron/electron/blob/f5339e15b13cab7b217c7c53f58259c333218333/.circleci/config.yml#L555